.github/workflows/clp-rust-checks.yaml

```
@@ -50,5 +47,7 @@ jobs:
         run: "task lint:check-rust"
 
       - name: "Test"
+        # Skip tests on macOS since the test requires LocalStack
+        if: "matrix.os != 'macos-15'"
```

actionlint (1.7.8)

property "os" is not defined in object type {}

(expression)